src/main/java/emu/grasscutter/game/inventory/GameItem.java

```diff
@@ -1,18 +1,6 @@
 package emu.grasscutter.game.inventory;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-import org.bson.types.ObjectId;
-
-import dev.morphia.annotations.Entity;
-import dev.morphia.annotations.Id;
-import dev.morphia.annotations.Indexed;
-import dev.morphia.annotations.PostLoad;
-import dev.morphia.annotations.Transient;
-
+import dev.morphia.annotations.*;
```

Comment:
Is this purely a lint change on an unrelated file?

Reply:
IDEA did this without my agreement. I'll consider fix the history when everything else's set

Reply:
> IDEA did this without my agreement. I'll consider fix the history when everything else's set

And all the 'bunch of files' mess. I'll force push to clean up to a mergable state after we reach an agreement.